src/default-theme/csvwidget.less

```diff
@@ -3,19 +3,46 @@
 | Distributed under the terms of the Modified BSD License.
 |----------------------------------------------------------------------------*/
 
-.jp-CSVWidget {
+.jp-CSVToolbarWidget {
+    display: flex;
+    flex-direction: row;
+    border-bottom: 1px solid #E0E0E0;
+    height: 24px;
+}
+
+.jp-CSVToolbarWidget select {
+    background: #FFFFFF;
+}
+
+.jp-CSVToolbarWidget .jp-CSVToolbarDropdown {
+    flex: 0 0 auto;
+    padding-left: 8px;
+    padding-right: 8px;
+    margin-left: 7px;
+    vertical-align: middle;
+    border: 1px solid #BDBDBD;
+    border-radius: 0;
+    outline: none;
+    width: 70px;
+    font-size: 14px;
```

Comment:
I tool this from the NoteBook's select box for cell type - https://github.com/jupyter/jupyterlab/blob/88c5b1e9823daac806f015f7cdc273c244819239/src/default-theme/notebook.less#L224

Should this be changed there too ?